### DISCUSSION TITLE
Email setup required in documentation and automation

### DISCUSSION THREAD

**Discussion by devhawkio** [logchimp/logchimp#251] [General]
There is no clear documentation or automation around email server and account config.
Here is an example:
`  "mail": {
    "host": "EMAIL_HOST",
    "port": PORT_NO,
    "user": "EXAMPLE_USER",
    "password": "EXAMPLE_PASS"
  }`

This must be added to the logchimp.config.json in order to send verification emails.

**Comment by mittalyashu** [comment]
@devhawkio 

What kind of automation are you looking for?

Using [LogChimp CLI](https://github.com/logchimp/logchimp-cli) you can easily setup a site, though, a newer version haven’t released yet as it still work in progress

**Comment by devhawkio** [comment]
I’m recommending automation in the CLI for the email server connection. I had to research the error message and find out how to create the connection on my own to make it work.

Thank you,
Bobby Lansing
CEO, DEVHAWK
________________________________

**Reply by mittalyashu** [reply]
I understand, will look into it.

Thanks
